downstream/modules/aap-hardening/ref-interactive-session-timeout.adoc

```diff
@@ -0,0 +1,24 @@
+[id="ref-interactive-session-timeout"]
+
+=  Interactive session timeout
+
+A compliance profile might require that an interactive session timeout be enforced. 
+For example, the DISA STIG requires that all users be automatically logged out after 15 minutes of inactivity. 
+The installation process often requires an hour or more to complete, and this control can stop the installation process and log out the user before installation is complete. 
+The same also applies to day two operations such as backup and restore, which in production environments often take longer than the recommended interactive session timeout. 
+During these operations, the interactive session timeout should be increased to ensure the operation is successful.
+
+There are multiple ways in which this control may be enforced, including shell timeout variables, setting the idle session timeout for `systemd-logind`, or setting SSH connection timeouts, and different compliance profiles can use one or more of these methods. 
+The one that most often interrupts the installation and day two operations is the idle session timeout for `systemd-logind`, which was introduced in the DISA STIG version V2R1 ({RHEL} 8) and V2R2 ({RHEL} 9). To increase the idle session timeout for `systemd-logind`, as the root user:
+
+* Edit the file `/etc/systemd/logind.conf`.
+* If the `StopIdleSessionSec` setting is set to zero, no change is needed.
+* If the `StopIdleSessionSec` setting is non-zero this indicates that the session will be terminated after that number of seconds. 
++
+Change `StopIdleSessionSec=7200` to increase the timeout, then run `systemctl restart systemd.logind` to apply the change.
+* Log out of the interactive session entirely and log back in to ensure the new setting applies to the current login session.
```

Comment:
Suggest changing "ensure" to "verify" or "make sure", per the IBM style guide. There are a few instances of this throughout your PR, so it would be good to do a global search/replace.

Reply:
My version says `Use "ensure" to refer to making sure, certain, or safe.`